framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioImpl.java

```diff
@@ -93,10 +93,8 @@ protected void deployKieDeployments() {
         project.processTemplateAndCreateResources(OpenShiftTemplate.CLUSTERED_CONSOLE_SMARTROUTER_TWO_KIE_SERVERS_TWO_DATABASES.getTemplateUrl(), envVariables);
 
         workbenchRuntimeDeployment = createWorkbenchRuntimeDeployment(project);
-        workbenchRuntimeDeployment.scale(1);
 
         smartRouterDeployment = createSmartRouterDeployment(project);
-        smartRouterDeployment.scale(1);
 
         kieServerOneDeployment = createKieServerDeployment(project, "1");
```

Comment:
In this test class is also used scale for KIE Servers. Please take a look if they can be also removed.

Reply:
I will remove it in here and rerun the whole suite just to be sure I don't break anything.